app/Http/Controllers/Web/ImportFileController.php

```diff
@@ -44,19 +44,17 @@ public function create($importType)
         }
 
         $data = [];
-        $userId = \Auth::user()->northstar_id;
-        $user = gateway('northstar')->asClient()->getUser('id', $userId);
 
         if ($importType === ImportType::$rockTheVote) {
+            $userId = \Auth::user()->northstar_id;
+            $user = gateway('northstar')->asClient()->getUser($userId);
+
             $data = [
-                'addr_street1' => $user->addr_street1,
-                'addr_street2' => $user->addr_street2,
+                'display_name' => $user->display_name,
                 'addr_city' => $user->addr_city,
                 'addr_zip' => $user->addr_zip,
-                'email' => $user->email,
-                'first_name' => $user->first_name,
-                'last_name' => $user->last_name,
-                'phone' => $user->mobile,
+                'email_preview' => $user->email_preview,
+                'phone_preview' => $user->mobile_preview,
```

Comment:
This makes the Test Import form slightly less convenient, since our logged in user would need to manually enter all of their user information. In general though, this is mostly used by me to test adding posts for new users, so I'll append a datetime to my email, and clear out the mobile (otherwise Chompy will find my user by mobile, and creating new user won't work)

The biggest inconvenience however would be the changes that remove the PII from the import result in `ImportRockTheVoteRecord.php`. We use these to display the user info after submitting the test form:
https://github.com/DoSomething/chompy/blob/9a7e8c13b154ee0a413e5f542a05e6d2dd78369c/app/Http/Controllers/Web/ImportFileController.php#L179-L186

You'd have to remember which unique email you used in order to update a post.

This PII is only used for the internal use of this test import form.. what do you think about keeping it in, per the use cases above?

Reply:
Ah, that's super helpful to know! Yeah, I'm good with keeping this in for that use-case.

Reply:
I've re-added these fields & updated the Northstar API call to include them in 76c08f3.